maven/bnd-resolver-maven-plugin/README.md

```diff
@@ -0,0 +1,36 @@
+# bnd-resolver-maven-plugin
+
+The `bnd-resolver-maven-plugin` is a bnd based plugin to resolve bundles from bndrun files.
+
+## What does the `bnd-resolver-maven-plugin` do?
```

Comment:
Since this needs to be invoked "by hand", perhaps you should include a command line example of running it?